lib/go-atscfg/remapdotconfig.go

```diff
@@ -244,10 +232,7 @@ func BuildRemapLine(cacheURLConfigParams map[string]string, atsMajorVersion int,
 			if _, globalExists := cacheURLConfigParams["location"]; globalExists {
 				log.Warnln("Making remap.config for Delivery Service '" + ds.Name + "': qstring_ignore == 1, but global cacheurl.config param exists, so skipping remap rename config_file=cacheurl.config parameter")
 			} else {
-				qstr, addedCacheURL, addedCacheKey := GetQStringIgnoreRemap(atsMajorVersion)
-				if addedCacheURL {
-					hasCacheURL = true
-				}
+				qstr, _, addedCacheKey := GetQStringIgnoreRemap(atsMajorVersion)
```

Comment:
If you're dropping support for cacheurl, you're dropping support for ATS < 6. GetQStringIgnoreRemap will need to be modified to return a static string.

Reply:
I didn't realize we still supported ATS < 6 in general